### PR TITLE
docs: small fixes in documentation

### PR DESCRIPTION
**Pull request by pim-simons**
Some small fixes in the docs we ran into

### CODE REVIEW

File: dashboard/installation/pipelines/dashboard.release.yaml [hunk 1]
```diff
@@ -49,7 +49,7 @@ stages:
             displayName: Download Artifact
           - template: ./Templates/dashboard-release-steps.yaml
             parameters:
-              azureSubscription: $(Infra.Environment.ServiceConnection)
+              azureSubscription: 'YourAzureSubscription'
```
Comment (jdefesche1) [inline dashboard/installation/pipelines/dashboard.release.yaml:52]: NameOfYourServiceConnection is better I think

File: dashboard/installation/pipelines/dashboard.release.yaml [hunk 2]
```diff
@@ -70,7 +70,7 @@ stages:
...
```
Comment (jdefesche1) [inline dashboard/installation/pipelines/dashboard.release.yaml:73]: dito

File: dashboard/installation/pipelines/dashboard.release.yaml [hunk 3]
```diff
@@ -92,7 +92,7 @@ stages:
...
```
Comment (jdefesche1) [inline dashboard/installation/pipelines/dashboard.release.yaml:95]: dito

File: dashboard/installation/pipelines/dashboard.release.yaml [hunk 4]
```diff
@@ -113,6 +113,6 @@ stages:
...
```
Comment (jdefesche1) [inline dashboard/installation/pipelines/dashboard.release.yaml:116]: dito

File: framework/installation/pipelines/framework.release.yaml [hunk 5]
```diff
@@ -49,7 +49,7 @@ stages:
             displayName: Download Artifact
           - template: ./Templates/framework-release-steps.yaml
             parameters:
-              azureSubscription: $(Infra.Environment.ServiceConnection)
+              azureSubscription: 'YourAzureSubscription'
```
Comment (jdefesche1) [inline framework/installation/pipelines/framework.release.yaml:52]: dito

File: dashboard/installation/pipelines/dashboard.release.yaml (hunk 1 above)
Comment (pim-simons) [reply dashboard/installation/pipelines/dashboard.release.yaml]: Changed in [`e718f9d` (#138)](https://github.com/invictus-integration/docs-ifa/pull/138/commits/e718f9d3a6d85baabe8033c134d1e69d4b502033)

**Review by grik001 — APPROVED** [review]

**Pull request merged**